modules/key-protect/outputs.tf

```diff
@@ -3,17 +3,22 @@
 # Copyright 2021 IBM
 #########################################################################################
 output "kms_key_crn" {
-  value = ibm_kms_key.key.id
+  value       = ibm_kms_key.key.id
+  description = "kms key crn"
```

Comment:
Could we add full sentences for the descriptions.

Reply:
Added descriptions to both input & output variables at examples & module level.